gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -509,20 +510,18 @@ MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 			_memorySubSpaceAllocate = _memorySubSpaceEvacuate;
 			_memorySubSpaceEvacuate = _memorySubSpaceSurvivor;
 			getMemorySpace()->setDefaultMemorySubSpace(getDefaultMemorySubSpace());
-			if (debug) {
-				omrtty_printf("tilt backout _allocateSpaceBase/Top %llx/%llx _survivorSpaceBase/Top %llx/%llx tilt sizes %llx %llx\n",
-						_allocateSpaceBase, _allocateSpaceTop, _survivorSpaceBase, _survivorSpaceTop,
-						(uintptr_t)_allocateSpaceTop - (uintptr_t)_allocateSpaceBase +  (uintptr_t)_survivorSpaceTop - (uintptr_t)_survivorSpaceBase, (uintptr_t)0);
-			}
+			Trc_MM_MSSSS_flip_backout(env->getLanguageVMThread(),
+					"",
+					_allocateSpaceBase, _allocateSpaceTop, _survivorSpaceBase, _survivorSpaceTop,
+					(uintptr_t)_allocateSpaceTop - (uintptr_t)_allocateSpaceBase +  (uintptr_t)_survivorSpaceTop - (uintptr_t)_survivorSpaceBase, (uintptr_t)0);
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

> Start the parameter list on a new line. Insert a newline immediately after the opening bracket.

This applies to all other instances in this PR.

```suggestion
			Trc_MM_MSSSS_flip_backout(
					env->getLanguageVMThread(),
					"",
					_allocateSpaceBase, _allocateSpaceTop, _survivorSpaceBase, _survivorSpaceTop,
					(uintptr_t)_allocateSpaceTop - (uintptr_t)_allocateSpaceBase +  (uintptr_t)_survivorSpaceTop - (uintptr_t)_survivorSpaceBase,
					(uintptr_t)0);
```